lib/elixir/src/elixir_interpolation.erl

```diff
@@ -78,9 +78,23 @@ unescape_tokens(Tokens) ->
   unescape_tokens(Tokens, fun unescape_map/1).
 
 unescape_tokens(Tokens, Map) ->
-  [unescape_token(Token, Map) || Token <- Tokens].
+  try [unescape_token(Token, Map) || Token <- Tokens] of
```

Comment:
We should need a try/catch  only on `unescape_tokens` as `unescape_token` itself is private.

Reply:
@josevalim Oh! thank you for bearing with me on all of these erlang changes! 😬 i'll update.